src/lib/storage/table.cpp

```diff
@@ -212,4 +212,12 @@ size_t Table::estimate_memory_usage() const {
   return bytes;
 }
 
+bool Table::is_validated() const { return _is_validated; }
```

Comment:
This will always be the same as all_of(chunks, chunk -> !chunk.has_mvcc_data()), right? Maybe make that a debug assert so these two "states" cannot diverge?

Reply:
Not that simple, right? Actually there are three possible states: Validated, NotValidated and NoMvccDataInTheFirstPlace.

Reply:
Exactly, see #1344. The idea here would be for this to simply be a "Validate has seen this table" flag.